.github/ISSUE_TEMPLATE.md

```diff
@@ -12,13 +12,15 @@ are using. If possible, please provide a test case or sample application that re
 the problem. This makes it much easier for us to diagnose the problem and to verify that
 we have fixed it.
 
-🚨 Security Vulnerability
-STOP!! Please don't raise security vulnerabilities here. Head over to https://pivotal.io/security to learn how to disclose them responsibly.
-
 🎁 Enhancement
 Please start by describing the problem that you are trying to solve. There may already
 be a solution, or there may be a way to solve it that you hadn't considered.
 
-🙅 Managed dependency upgrade
-You DO NOT need to raise an issue for a managed dependency version upgrade as there's a semi-automatic process for checking managed dependencies for new versions before a release. But pull requests for upgrades that are more involved than just a version property change are still most welcome.
+🙅 "Please DO NOT Raise an Issue" Cases
```

Comment:
Shouldn't question be in this category? If you agree, I wonder if that shouldn't be on top (let's increase our change that users actually read that).